lib/td/command/export.rb

```diff
@@ -48,6 +49,10 @@ def table_export(op)
       raise ArgumentError, "#{s} is not a supported encryption method" unless SUPPORTED_ENCRYPT_METHOD.include?(s)
       encryption = s
     }
+    op.on('-a', '--assume-role SESSION_NAME#ROLE_ARN', 'export with assume role with SESSION_NAME as session name and ROLE_ARN as role arn') {|s|
+      raise ArgumentError, "#{s} is not a supported assume role style. Please add ROLE_ARN after SESSION_NAME#" unless s.include?('#')
```

Comment:
I checked [AWS's manual](http://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html) and it says the session name is [\w+=,.@-]{2,64}. So you can separate RoleSessionName and RoleArn with other than those.

Anyway I noticed some example of [AWS CLI's user guide](http://docs.aws.amazon.com/cli/latest/userguide/cli-roles.html) doesn't specify session name. It automatically specify RoleSessionName. Maybe td CLI should also follow [AWS CLI's options](http://docs.aws.amazon.com/cli/latest/reference/sts/assume-role.html) and RoleSessionName is optional (or completely drop for UI).